engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -227,6 +227,8 @@ def __init__(self, **kwargs):
             self.output_formatter = _jsonlines_output_formatter
         elif "none" == formatter:
             pass
+        elif callable(formatter):
```

Comment:
How we pass this from `serving.properties`? Where is the code load this callable?

Reply:
user cannot pass this through serving.properties, since bunch of the postprocessing logic is not descriptable. This option is just trying to make it easy for customer to extend from the base class

Reply:
We should be able load the module by itself name